src/ResourceManager/Common/Commands.Common.Strategies.UnitTest/TimeSlotTest.cs

```diff
@@ -0,0 +1,72 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace Microsoft.Azure.Commands.Common.Strategies.UnitTest
+{
+    [TestClass]
```

Comment:
These should be xunit tests, and we will also need entries for them in the testmap

Reply:
I've converted the project to XUnit.